Spigot-Server-Patches/0549-Fix-harming-potion-dupe.patch

```diff
@@ -0,0 +1,61 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: PepperCode1 <44146161+PepperCode1@users.noreply.github.com>
+Date: Thu, 23 Jul 2020 14:25:07 -0700
+Subject: [PATCH] Fix harming potion dupe
+
+EntityLiving#applyInstantEffect() immediately kills the player and drops their inventory.
+Before this patch, instant effects would be applied before the potion ItemStack is removed and replaced with a glass bottle. This caused the potion ItemStack to be dropped before it was supposed to be removed from the inventory. It also caused the glass bottle to be put into a dead player's inventory.
+This patch makes it so that instant effects are applied after the potion ItemStack is removed, and the glass bottle is only put into the player's inventory if the player is not dead. Otherwise, the glass bottle is dropped on the ground.
+
+diff --git a/src/main/java/net/minecraft/server/ItemPotion.java b/src/main/java/net/minecraft/server/ItemPotion.java
+index 0d204a46b8b23b6fb58acffc5de085e15f77767d..7dfd0691c89bdddcbccfe19911d91e9aaf0c64d1 100644
+--- a/src/main/java/net/minecraft/server/ItemPotion.java
++++ b/src/main/java/net/minecraft/server/ItemPotion.java
+@@ -2,6 +2,7 @@ package net.minecraft.server;
+ 
+ import java.util.Iterator;
+ import java.util.List;
++import java.util.ArrayList; // Paper - Fix harming potion dupe
```

Comment:
```suggestion
```

Reply:
The import has been removed in the latest version.